ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -445,6 +455,87 @@ Loader {
         }
     }
 
+    Component {
+        id: deletedMessageComponent
+
+        ColumnLayout {
```

Comment:
This `ColumnLayout` seems redundant

Reply:
Maybe I'm just a QML noob, but without it, the `Layout.` properties in the RowLayout didn't work. I think it's because the parent of `deletedMessageComponent`'s Loader is not a Layout.

Reply:
Yeah... it's weird to have `Loader` as the toplevel item, you never know in which context the component ends up being used

Reply:
Ok I see, it indeed will do the trick here. It's not right, but shouldn't hurt, so I guess we can keep it as it will be rewritten anyway 🙂